uktrade/data-hub-frontend
# Create a business details tab on Company page

## Description of change

On the companies tab page we created the Business Details tab and added the content from the Business Details page (example: https://www.datahub.trade.gov.uk/companies/689f42c1-a098-e211-a939-e4115bead28a/business-details). 

The ‘View full business details’ link in the header has also need to be removed.

This will ensure consistency in how the information is displayed as tabs.

## Test instructions

-Visit a company e.g. http://localhost:3000/companies/0eb9e12e-2591-483d-a677-3e54ea721f3b/activity
-You should see a Business details tab instead of a link to View business details 

## Screenshots

### Before

<img width="1015" alt="image" src="https://user-images.githubusercontent.com/83657534/220667403-7f73310b-7871-4764-9fd4-5282866fccfd.png">


### After
![image](https://user-images.githubusercontent.com/83657534/220875064-15cd2033-16da-4b75-a93b-56b799802ec0.png)

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
